plugins/woocommerce/tests/e2e-pw/tests/merchant/create-woocommerce-blocks.spec.js

```diff
@@ -169,28 +169,42 @@ test.describe(
 
 					// eslint-disable-next-line playwright/no-conditional-in-test
 					if ( blocks[ i ] === 'Reviews by Product' ) {
+						// Use click() instead of check().
+						// check() causes occasional flakiness:
+						//     - "Error: locator.check: Clicking the checkbox did not change its state"
 						await canvas
 							.locator( '.wc-block-reviews-by-product' )
 							.getByLabel( simpleProductName )
-							.check();
+							.click();
 						await canvas
 							.locator( '.wc-block-reviews-by-product' )
 							.getByRole( 'button', {
 								name: 'Done',
 								exact: true,
 							} )
 							.click();
+						// Click on the Reviews by Product block to show the Block Tools to be used later.
+						await canvas
+							.getByLabel( 'Block: Reviews by Product' )
+							.click();
```

Comment:
I had to add this step because the "Reviews by Product" block doesn't show the Block Tools at the end of the previous step, which is inconsistent with how the insertion of the other blocks ends -- they all show the Block tools at the end of their insertion, "Reviews by Product" block doesn't. 

Showing the Block Tools is crucial in order to prepare for the insertion of the next block in the list.